### DISCUSSION TITLE
Windows 10 EOL

### DISCUSSION THREAD

**Discussion by mansellan** [twinbasic/twinbasic#227] [General]
Microsoft has previously committed to supporting the VB6 runtime for the lifetime of Windows 10. As Win10 was [supposed to be the "last version of Windows"](https://www.theverge.com/2015/5/7/8568473/windows-10-last-version-of-windows), this implied support in perpetuity.

However, it seems this has changed. Windows 11 is [rumored to be on its way](https://arstechnica.com/gadgets/2021/06/make-way-for-windows-11-windows-10-end-of-life-is-october-2025/), and Microsoft seem set to [retire Windows 10 in 2025  ](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro). I would be amazed if the VB6 runtime is supported in Win11.

So, twinBasic appears to offer (a? the only?) lifeline for VB6 projects. No pressure @WaynePhillipsEA!

**Comment by dmrvb** [comment]
My guess is that Microsoft will continue support the VB runtime.
If they choose not too then TwinBasic could be a real money spinner for 
Wayne.
Worse case is they remove all support for compiled code and make it 
managed code only, at that point I will likely stop writing software.

.....Dave

On 16/06/2021 21:00, Andrew Mansell wrote:
> Microsoft has previously committed to supporting the VB6 runtime for the 
> lifetime of Windows 10. As Win10 was supposed to be the "last version of 
> Windows" 
> <https://www.theverge.com/2015/5/7/8568473/windows-10-last-version-of-windows>, 
> this implied support in perpetuity.
> 
> However, it seems this has changed. Windows 11 is rumored to be on its 
> way 
> <https://arstechnica.com/gadgets/2021/06/make-way-for-windows-11-windows-10-end-of-life-is-october-2025/>, 
> and Microsoft seem set to retire Windows 10 in 2025 
> <https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro>. 
> I would be amazed if the VB6 runtime is supported in Win11.
> 
> So, twinBasic appears to offer (a? the only?) lifeline for VB6 projects. 
> No pressure @WaynePhillipsEA <https://github.com/WaynePhillipsEA>!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/WaynePhillipsEA/twinbasic/discussions/227>, or 
> unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AT752Y54HWOUX52TXZ55LK3TTD7GTANCNFSM462FCVOQ>.
> 

-- 
-----------------------------------------------

Dave Rhodes

Bespoke Software, Instrumentation and Technology for
Engineering Sound and Vibration.


------------------------------------------------

-- 
This email has been checked for viruses by AVG.
https://www.avg.com
